README.md

```diff
@@ -6,32 +6,16 @@
 
 # Music Releases
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## Getting Started with the Project
-
-### Dependency Installation & Startup Development Server
-
-Once cloned, navigate to the project's root directory and this project uses npm (Node Package Manager) to manage its dependencies.
-
-The command below is a combination of installing dependencies, opening up the project on VS Code and it will run a development server on your terminal.
-
-```bash
-npm i && code . && npm run dev
-```
+The assignment was to use React to create components containing information about albums from spotify, with links to artist page and album page. 
+Each album should have a hover effect with buttons, and be placed in a grid that is responsive to screen.
 
 ### The Problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I struggled with understanding the data flow in react. I spent a lot of time going through how to use props to send data to each component, chat gpt helped me a lot here. I also struggled some with getting the styling right, getting the icons to grow but stay in the exact same position, then a class mate suggested to use scale, very helpful. 
+I also stuggles with getting the links of the artist name right when there was more than one, but i finally got it.
```

Comment:
I'm really impressed, you did it!! This really was a tricky part of the assignment, right?!